action.yml

```diff
@@ -1,9 +1,6 @@
 name: 'Azure SQL Deploy'
 description: 'Deploy a DACPAC or a SQL script to Azure SQL database'
 inputs:
-  server-name:
-    description: 'Name of the Azure SQL Server name, like Fabrikam.database.windows.net.'
-    required: true
```

Comment:
Please set required:false
As we shouldn't break the existing workflows where users might be passing the server name explicitly